### DISCUSSION TITLE
Prediction Score and Labelling the corners of the arena

### DISCUSSION THREAD

**Discussion by druoz** [talmolab/sleap#1219] [Help!] [answered]
Hello all,
I'm new on SLEAP and I'm tracking 1 hen and 3 chicks in a rectangular shed. 
I want to measure the distance between the animals and the distance between the animals and the shed corners.

So my first question: how can label the corners? (I labeled corner points as an animal but ı am not sure that is correct?)
Question 2: What is the prediction score? And How can I find out the index of certainly/likelihood of tracking for each markers at each frame?

I would really appreciate if you could reply. 
Thanks in advance.

**Comment by roomrys** [comment, marked as answer]
Hi @druoz,

You can definitely label the corners as an instance/"animal"!  We have had cases where people have used a "monster" skeleton which tracks both the animal and the cage using a single project/skeleton. We have also had cases where people have created two separate projects/skeletons - one to track the animal and one to track the environment. I would recommend the latter, but you can definitely try and see which works best for you. Note that SLEAP only supports one skeleton per project at the moment, so you will need multiple projects to handle multiple skeletons.

The prediction score is how confident SLEAP is that the prediction is correct. Usually, the higher the score, the more accurate the prediction. The calculation for the mean frame score is shown below where `inst.score` is the prediction score of each instance/"animal":
https://github.com/talmolab/sleap/blob/4bd017150acad47e7586c891fe9855eef0b4c48f/sleap/gui/dataviews.py#L511-L519

Tracking is performed after pose estimation and is based on distance between sequential instances (i.e. we need good pose estimation on all frames before we perform tracking). SLEAP v1.3.0 (not yet released) will have identity-based tracking which uses visual features to identify different animals (and can be done concurrently with pose estimation).

Let me know if that answered you questions and/or if more questions are raised.

Thanks,
Liezl

**Reply by druoz** [reply]
Hi Liezl,
Thank you for response.
Regarding accuracy data:
What is the unit of measurement of accuracy for point, instances and tracking accuracy?
In my case, accuracy score for tracking ranges 0-1, instance scores above 5 (does this depend on number of body parts, is this  a sum?) and point score above 1. Can you explain the range?
Thanks in advance.
Ugur

**Comment by druoz** [comment]
Hello,
After exporting data, I see three different accuracy score.
What is the unit of measurement of accuracy for point, instances and tracking accuracy?
In my case, accuracy score for tracking ranges 0-1, instance scores above 5 (does this depend on number of body parts, is this a sum?) and point score above 1. Can you explain the range?
I would be really appreciate if you could reply.
Thanks in advance. 
Ugur

**Reply by roomrys** [reply]
Followed up in #1268.
